README.md

```diff
@@ -68,10 +68,8 @@ var api = new ParseServer({
 // Serve the Parse API on the /parse URL prefix
 app.use('/parse', api);
 
-// Hello world
-app.get('/', function(req, res) {
-  res.status(200).send('Express is running here.');
-});
+// Serve files from the parse/public directory 
+app.use(express.static(__dirname + '/parse/public'));
```

Comment:
thanks @gfosco for the response! 

I don't recall doing anything special in the configuration of my parse project, and if i just do `app.use(express.static(__dirname + '/public')); `
it says "Cannot GET /" because my directory structure is like 

`
/
- node_modules
- parse
-- cloud
-- config
-- public
`
maybe this is unique to my two parseapps though... if you think that is the case and you would like me to change it to /public please reply again and i will make that edit.

Thanks for reviewing!!